compute_endpoint/globus_compute_endpoint/endpoint/config/default_config.yaml

```diff
@@ -1,4 +1,4 @@
-display_name: None
+display_name: null
```

Comment:
We should probably just remove `display_name` from the default config.

Reply:
Original reasoning is display_name is a feature field, one of the few most might want to modify.

Let's discuss in scrum whether it should be removed.